prometheus_client/openmetrics/exposition.py

```diff
@@ -136,8 +140,12 @@ def generate_latest(registry, escaping=UNDERSCORES):
                             nh_exemplarstr = _compose_exemplar_string(metric, s, nh_ex)
                             exemplarstr += nh_exemplarstr
 
+                # Skip native histogram samples entirely if version < 2.0.0
+                if s.native_histogram and Version(version) < Version('2.0.0'):
```

Comment:
Couldn't we do this skip on line 95 and then not have to add `and Version(version) >= Version('2.0.0')` in other places?

Reply:
or maybe even further up?